SlowCheetah.JDT/Processors/JdtDefault.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Sayed Ibrahim Hashimi. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See  License.md file in the project root for full license information.
+
+namespace SlowCheetah.JDT
+{
+    using System.Linq;
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// Represents a recursive JDT transformation
+    /// </summary>
+    internal class JdtDefault : JdtProcessor
+    {
+        /// <inheritdoc/>
+        public override string Verb { get; } = null;
+
+        /// <inheritdoc/>
+        public override void Process(JObject source, JObject transform)
+        {
+            // JDT Verbs are not handled here
+            foreach (JProperty transformNode in transform.Properties()
+                .Where(p => !JsonUtilities.IsJdtSyntax(p.Name)))
+            {
+                JToken nodeToTransform;
+                if (source.TryGetValue(transformNode.Name, out nodeToTransform))
+                {
+                    // If the node is present in both transform and source, analyze the types
+                    // If both are objects, that is a recursive transformation, not handled here
+                    if (nodeToTransform.Type == JTokenType.Array && transformNode.Value.Type == JTokenType.Array)
+                    {
+                        // If the original and transform are arrays, merge the contents together
+                        JsonUtilities.MergeArray((JArray)nodeToTransform, (JArray)transformNode.Value);
+                    }
+                    else if (nodeToTransform.Type != JTokenType.Object || transformNode.Value.Type != JTokenType.Object)
+                    {
+                        // TO DO: Verify if object has JDT verbs
```

Comment:
We should possibly generate an error if the default transformation (and the other transformations as well) results in an addition or replacement of a node with an object that contains verbs. Those verbs are being processed because we don't recurse into the node.